Sources/SwiftRoaring/RoaringBitmap.swift

```diff
@@ -8,9 +9,12 @@ public typealias RoaringStatistics = roaring_statistics_t
 ///
 /// Swift wrapper for CRoaring (a C/C++ implementation at https://github.com/RoaringBitmap/CRoaring)
 ///
-public class RoaringBitmap: Sequence, Equatable, CustomStringConvertible,
-                            Hashable, ExpressibleByArrayLiteral {
+public final class RoaringBitmap: Sequence, Equatable, CustomStringConvertible,
+                                  Hashable, ExpressibleByArrayLiteral, SetAlgebra, Codable {
+
+    @usableFromInline
```

Comment:
I am not especially happy about that. The attribute `ptr` would be best left private/hidden unless we have very specific reasons to expose it.

Similarly, functions below that were previously `private` are made `public`. I think that this requires justification. By default, I would not do that.

Reply:
It's not public, it's internal (I can make that explicit). The reason for that is that so that it's usable from inline (private can't).

Reply:
Ok.